eng/versioning/version_client.txt

```diff
@@ -60,7 +60,7 @@ com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-tests;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-test-core;2.3.2;2.3.3-beta.1
-com.microsoft.azure:spring-data-cosmosdb;3.0.0-beta.1;3.0.0-beta.1
+com.azure:azure-spring-data-cosmos;3.0.0-beta.1;3.0.0-beta.1
```

Comment:
nit: Please put this alphabetically in the file. With the groupId change this should go in between com.azure:azure-sdk-template and com.azure:azure-storage-blob

Reply:
Done.